cli/standalone/binary.rs

```diff
@@ -620,9 +620,17 @@ impl<'a> DenoCompileBinaryWriter<'a> {
     };
     for include_file in include_files {
       let path = deno_path_util::url_to_file_path(include_file)?;
-      vfs
-        .add_file_at_path(&path)
-        .with_context(|| format!("Including {}", path.display()))?;
+      if path.is_dir() {
+        // todo(dsherret): we should analyze if any of these are
+        // modules in order to include their dependencies
+        vfs
```

Comment:
That's totally fair for a first pass. Can you please open an issue about it before landing?

Reply:
https://github.com/denoland/deno/issues/26941